### DISCUSSION TITLE
Table format

### DISCUSSION THREAD

**Discussion by sgvignali** [CREWdecisions/elicitr#15] [General]
@canessas We must define the structure of the table generated by Google Sheets. I think that we always have something like this:

- Timestamp, always as first column
- Name, always as second column
- Estimates, several columns, depending on the topic
- Confidence, always last column

Basically, first, second and last column are fixed, and we can leave the number of estimates flexible.

Thoughts? 🤔

**Comment by canessas** [comment]
I think the structure is timestamp - name - variable 1 (min) - variable 1 (max) - variable 1 (mode) - variable 1 (confidence) - variable 2 ( min) - variable 2 (max) - etc

So each variable always has four values, min-max-mode-confidence
There could be more variables one after the other

**Reply by sgvignali** [reply]
Ok, yes, it make sense!

**Reply by canessas** [reply]
we tend to ask for it in this specific sequence min - max - mode - confidence because it's been proven to reduce bias
But of course some people then make mistakes and for example write modes that are outside the min-max range... so I normally just switch them

**Reply by sgvignali** [reply]
This is going to be difficult to do programmatically... or maybe not. But then you assume they enter the value in the wrong cell?
